### DISCUSSION TITLE
在快速录入添加【库存下限】

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#10] [想法]
在快速录入添加【库存下限】

**Comment by densen2014** [comment]
安排

**Comment by densen2014** [comment]
升级到 2023-1-10 版本

快速录入优化: 添加'库存下限', 完善商品资料填充, 

![image](https://user-images.githubusercontent.com/8428709/211586116-ddef2630-26be-4168-8e9b-53db6129b119.png)

**Comment by aaronameixa** [comment]
如何升级到 2023-1-10 版本？

**Reply by densen2014** [reply]
加微信 m916423330,  旧版升级最好问一下客服, 2022 可以直接安装  https://file2.app1.es/POS/230110.exe

**Comment by m916423330** [comment]
测试效果ok
